Sasquatch/SasquatchPuppet/GoogleService-Info.plist

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>177539951155-e7ru6edkes3evhskg75oh68qjgvc2f7f.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.177539951155-e7ru6edkes3evhskg75oh68qjgvc2f7f</string>
+	<key>ANDROID_CLIENT_ID</key>
+	<string>177539951155-nre1cirjftpfdpt4pf21749j00vbbj4s.apps.googleusercontent.com</string>
+	<key>API_KEY</key>
+	<string>AIzaSyCHca6DXwzjIw5QDvQI4Kgy1dn6StCkD2w</string>
```

Comment:
Remove the key and rollover on Google console.

Reply:
Removed.